README.md

```diff
@@ -24,9 +24,10 @@ git clone https://github.com/libjxl/libjxl.git --recursive
 ```
 
 This repository uses git submodules to handle some third party dependencies
-under `third_party/`, that's why is important to pass `--recursive`. If you
+under `third_party`, that's why is important to pass `--recursive`. If you
 didn't check out with `--recursive`, or any submodule has changed, run:
-`git submodule update --init --recursive`.
+
+    git submodule update --init --recursive
```

Comment:
add ``` and remove indent?

Reply:
I changed it to use the tick marks.  I also edited my recent document submissions (crossroad, msys) to also use tick marks.